frontend/src/static/js/v_summary_charts.js

```diff
@@ -31,6 +31,7 @@ window.vSummaryCharts = {
     },
 
     ...Vuex.mapState(['mergedGroups']),
+    ...Vuex.mapState(['fileTypeColors']),
```

Comment:
Can this statement be merged with the previous one?

Reply:
Yes, I have merged it.